src/main/java/org/spongepowered/common/registry/SpongeGameRegistry.java

```diff
@@ -397,7 +394,7 @@
     private final Map<String, CookedFish> cookedFishMappings = Maps.newHashMap();
     private final Map<String, DyeColor> dyeColorMappings = Maps.newHashMap();
     private final Map<String, Art> artMappings = Maps.newHashMap();
-    private final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
+    protected final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
```

Comment:
It's not very good I know but the Sponge mod has to use a different entity class, see [SpongeModGameRegistry:120](https://github.com/SpongePowered/Sponge/pull/207/files#diff-fc191454fb273d4eafe538517be30af5R120)